lib/directive.js

```diff
@@ -0,0 +1,26 @@
+var angular = require('angular');
+var linkifyElement = require('linkifyjs/element');
+
+angular.module('linkify').directive('linkify',[
+  'linkify',
+  function (linkify) {
+    var defaultOptions = linkify.getOptions() || {};
+
+    var link = function (scope, element) {
+      var options = angular.extend(defaultOptions, scope.options);
+
+      scope.$evalAsync(function () {
+        try {
```

Comment:
There's a bizarre error on their part where if it finds just one text node that's empty, it errors and breaks the module